index.html

```diff
@@ -101,6 +101,14 @@ <h3>
         aware of such actions, stateful interactions should be considered out
         of scope.
       </p>
+      <h3>
+        Subresource loading
+      </h3>
+      <p>
+        While many popular model formats support the use of external asset
+        requests for better re-use and consistency, this initial specification
```

Comment:
I would phrase it stronger:
this specification will only allow assets that are self-contained

Reply:
Perfect. updated!

Reply:
looks great